curriculum/challenges/english/25-front-end-development/quiz-html-accessibility/66ed9026f45ce3ece4053ebb.md

```diff
@@ -17,439 +17,446 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What are the best methods for creating accessible data tables? 
 
 #### --distractors--
 
-Placeholder distractor 1
+Add a caption by using the `<paragraph>` tag
 
 ---
 
-Placeholder distractor 2
+Use the `aria-label` attribute to make sure that the data cells associate with the proper header
 
 ---
 
-Placeholder distractor 3
+Use absolute sizing to make sure that the data cells are not too large
 
 #### --answer--
 
-Placeholder answer
+Designate row and/or column headers by using `<td>` and`<th>` tags
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the difference between the attributes `aria-label` and `aria-labelledby`?
 
 #### --distractors--
 
-Placeholder distractor 1
+The `aria-label`and `aria-labelledby` attributes serve the same purpose.
 
 ---
 
-Placeholder distractor 2
+The `aria-label` attribute is used to give an color to its label while the `aria-labelledby` attribute is used to give height to its label.
 
 ---
 
-Placeholder distractor 3
+The `aria-label` and `aria-labelledby` attributes keeps labels undefined. 
 
 #### --answer--
 
-Placeholder answer
+The `aria-labelledby` attribute is used to give an element a label while the `aria-label` attribute is used to give an element its label. 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the function of the `aria-selected` attribute? 
 
 #### --distractors--
 
-Placeholder distractor 1
+The function of the `aria-selected` attribute is to change the font size of an element.
 
 ---
 
-Placeholder distractor 2
+The function of the `aria-selected` attribute is to change the background color of an element.
 
 ---
 
-Placeholder distractor 3
+The function of the `aria-selected` attribute is to shrink an element.
 
 #### --answer--
 
-Placeholder answer
-
+The `aria-selected` attribute indicates if an element is selected.
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the function of `aria-expanded`? 
 
 #### --distractors--
 
-Placeholder distractor 1
+The function of `aria-expanded` is to change the font size of an element
 
 ---
 
-Placeholder distractor 2
+The function of `aria-expanded` is to change the background color of an element
 
 ---
 
-Placeholder distractor 3
+The function of `aria-expanded` is to to shrink an element 
 
 #### --answer--
 
-Placeholder answer
+The function of `aria-expanded` is to indicate whether a control is currently expanded or collapsed, and whether its associated content is displayed or hidden
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the correct way to associate a label with an input?
 
 #### --distractors--
 
-Placeholder distractor 1
+Use separate `label` and `input` elements without any association
 
 ---
 
-Placeholder distractor 2
+Use placeholder text instead of actual `label` elements for form inputs
 
 ---
 
-Placeholder distractor 3
+Use nested `div` elements to group labels and inputs together
```

Comment:
I'm not sure what this actually means. Can you give a code example?

Reply:
I was trying to think of an incorrect answer, but it didn't come the way I wanted.